src/lib.rs

```diff
@@ -86,6 +86,9 @@ extern crate std;
 
 extern crate untrusted;
 
+#[macro_use]
```

Comment:
This should also have `#[no_link]` right? That way we don't have to worry about the `native_versioning` code changing in a way where it affects the codegen besides the naming.

Reply:
> I think #[no_link] is a no-op here, and I can't see why it would affect anything in the future. But I'm happy to add it.

It seems like a no-op now. However, if in the future the `native_versioning` crate were to change so that this isn't a no-op, then the build would fail. And that's what I want.